module.txt

```diff
@@ -13,6 +13,10 @@
             "id": "FlexiblePathfinding",
             "minVersion": "1.0.0"
         },
+        {
+            "id": "Economy",
+            "minVersion": "1.1.0-SNAPSHOT"
```

Comment:
We should make a release of the current state of Economy and bump the version number ... :thinking: